py-polars/tests/unit/streaming/test_streaming_group_by.py

```diff
@@ -66,7 +66,7 @@ def test_streaming_group_by_types() -> None:
                         pl.col("bool").last().alias("bool_last"),
                         pl.col("bool").mean().alias("bool_mean"),
                         pl.col("bool").sum().alias("bool_sum"),
-                        pl.col("date").sum().alias("date_sum"),
+                        # pl.col("date").sum().alias("date_sum"),
```

Comment:
Was this intentional?

Reply:
@ritchie46 Yes, the date sum doesn't really make any sense, and it's not supported on the eager engine either.

Reply:
Right, shouldn't we error during IR resolving then?